src/DynamoCore/Models/PresetsModel.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Dynamo.Core;
+using Dynamo.Selection;
+using Dynamo.Nodes;
+using System.Xml;
+using Dynamo.Interfaces;
+
+namespace Dynamo.Models
+{
+    /// <summary>
+    /// a class that holds a set of preset design options states
+    /// there is one instance of this class per workspacemodel
+    /// </summary>
+    public class PresetsModel:ILogSource
+    {
+        #region private members
+        private readonly List<PresetState> designStates;
```

Comment:
You can initialize the collection right here.  Then, you can simply omit the constructor completely.